cmake/configure/configure_3_petsc.cmake

```diff
@@ -103,6 +103,15 @@ MACRO(FEATURE_PETSC_CONFIGURE_EXTERNAL)
   SET(DEAL_II_EXPAND_PETSC_BLOCKVECTOR "PETScWrappers::BlockVector")
   SET(DEAL_II_EXPAND_PETSC_MPI_VECTOR "PETScWrappers::MPI::Vector")
   SET(DEAL_II_EXPAND_PETSC_MPI_BLOCKVECTOR "PETScWrappers::MPI::BlockVector")
+  #
+  # FIXME:
+  # temporary variable until deal.II fully support complex-valued PETSc
+  IF(NOT PETSC_USE_COMPLEX)
```

Comment:
but then all the checks in those macros must be broken?! like `PETSC_VERSION_MAJOR` and `PETSC_WITH_MPIUNI`... @tamiko ?

i need that info to expand in `REAL_SERIAL_VECTORS` to know which vectors to use for instantiation. I can't rely on `#ifndef` in the actual code here...